diff --git a/fortran/UTILITY.FOR b/fortran/UTILITY.FOR
index 9351235..822d7de 100644
--- a/fortran/UTILITY.FOR
+++ b/fortran/UTILITY.FOR
@@ -2496,6 +2496,9 @@ c  double precision variables
       elseif (hvar.eq.'ttpn') then
         v=ttpn(icomp)
         if (l) ttpn(icomp)=dbl
+      elseif (hvar.eq.'ptpn') then
+        v=ptpn(icomp)
+        if (l) ptpn(icomp)=dbl
       elseif (hvar.eq.'txeos') then
         v=txeos(icomp)
         if (l) txeos(icomp)=dbl
